protocol/0057-TRAN-transfers.md

```diff
@@ -87,6 +87,41 @@ To be able to dispatch rewards to reward pools of multiple markets pro rata to t
 - `reward metric asset` — (the settlement asset of all markets that will be in scope for the transfer)
 - `market scope` — a subset of markets in which parties are eligible to be rewarded from this transfer.
   - If the market scope is not defined / an empty list, it is taken as all the markets that settle in the reward metric asset.
+
+A party can further control their recurring transfer funding the reward pool by defining the entities which are within scope. Entities within scope can be either individual parties or [teams](./0083-RFPR-on_chain_referral_program.md#glossary). When scoping individuals, a subset of keys can be detailed, and when scoping teams a specific set of team ids can be detailed.
+
+To support entity scoping, the transaction include the following fields:
+
+- `entity scope` - mandatory enum which defines the entities within scope.
+  - `ENTITY_SCOPE_INDIVIDUALS` - the rewards must be distributed directly amongst eligible parties
+  - `ENTITY_SCOPE_TEAMS` - the rewards must be distributed amongst directly eligible teams (and then amongst team members)
+- `individual scope` - optional enum if the entity scope is `ENTITY_SCOPE_INDIVIDUALS` which defines the subset of individuals which are eligible to be rewarded.
+  - `INDIVIDUAL_SCOPE_ALL` - all parties on the network are within the scope of this reward
+  - `INDIVIDUAL_SCOPE_IN_TEAM` - all parties which are part of a team are within the scope of this reward
+  - `INDIVIDUAL_SCOPE_NOT_IN_TEAM` - all parties which are not part of a team are within the scope of this reward
+- `team scope` - optional list if the reward type is `ENTITY_SCOPE_TEAMS`, field allows the funder to define a list of team ids which are eligible to be rewarded from this transfer
+- `staking_requirement` - the required minimum number of tokens staked for a party to be considered eligible. Defaults to `0`.
```

Comment:
I would imagine that  `staking_requirement` and `notional_time_weighted_average_position_requirement` are not applicable for the validator ranking metric. same comment about scope `ENTITY_SCOPE_TEAMS`

Reply:
what I meant is that I don't see how ENTITY_SCOPE_TEAMS is relevant for the validator ranking metric? this could only be relevant if the validator public keys are members in teams which I don't know if we want.

Reply:
Ah I'm with you. 

There is nothing in the referral spec stopping validators creating/joining teams so I would lean towards supporting this combination of ENTITY_SCOPE_TEAMS and validator reward metric. Also helps with consistency, ideally all metrics would work for both individual and team scopes.

If some complexity in the implementation comes up though we can re think it because using the individual scope is the more likely use case for this metric.

Reply:
same question on market proposer, in this case the reward is targeted to individuals, I'm not sure what it would mean to specify distribution strategy and scope teams for this metric.

Reply:
I think it harmless to support but since we now have two metrics where it might make sense to disable the `ENTITY_SCOPE_TEAMS` and `DISTRIBUTION_STRATEGY_RANK` it might be best to prioritise clarity over consistency.

I'll raise this in the protocol-design session now and let you know.

Reply:
@ze97286 - if the restrictions are preferred to be added then this should be done, no hard requirements from research